app/views/common/BreadcrumbComponent.vue

```diff
@@ -0,0 +1,36 @@
+<style scoped>
+    a {
+         color: #065e73;
+     }
+</style>
+
+<template>
+    <div class="breadcrumbs rtl-allowed">
+        <a v-if="links.length > 0" :href="links[0].href">{{ links[0].i18n ? $t(links[0].i18n) : links[0].text }}</a>
+        <span v-if="links.length > 1">
```

Comment:
These checks are not pretty. Especially the weirdly structured `v-if` because we are only rendering a link if there is an `href` property and text otherwise. Would love suggestions on how to clear this up so it is 1) more legible to humans and 2) cleaner Vue HTML syntax.

Reply:
I'm not completely understanding the problem, but I _think_ something like this would work?  Let me know if you want to discuss offline:

```
<ul class="breadcrumbs rtl-allowed">
  <li v-for="link in links">
    <a v-if="link.href">{{ link.text }}</a>
    <span v-else>{{ link.text }}</span>

    <span class="separator">&gt;</span>
  </li>
</ul>
```

We can hide the last separator with `:last-of-type`.  We could probably add the separator itself with `::after` or maybe even a list style?